.github/workflows/build.yaml

```diff
@@ -0,0 +1,51 @@
+name: build-refdocs
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+  pull_request:
+
+jobs:
+  build:
+    name: Build reference docs
+
+    strategy:
+      matrix:
+        go-version: [1.15.x]
+        platform: [ubuntu-latest]
+
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+
+      - name: Set up Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+        id: go
+
+      - name: Check out reference-docs code
+        uses: actions/checkout@v2
+
+      - name: Set release version
+        id: version_value
+        run: |
+            echo 'KUBE_VERSION<<EOF' >> $GITHUB_ENV
+            awk '/^version/ {print}' config.toml | cut -c 12-16 >> $GITHUB_ENV
+            echo 'EOF' >> $GITHUB_ENV
+
+      - name: Build references
+        run: |
+          echo "Building comp and gen-resourcesdocs"
+          echo "$GITHUB_WORKSPACE"
+          mkdir -p ${BUILD_PATH}/src/k8s.io
+          echo "${BUILD_PATH}"
+          wget https://github.com/kubernetes/kubernetes/archive/${KUBE_VERSION}.0.tar.gz -O ${BUILD_PATH}/src/k8s.io/kubernetes-src.tar.gz
+          pushd ${BUILD_PATH}/src/k8s.io && tar xzf kubernetes-src.tar.gz && mv kubernetes-${KUBE_VERSION}.0 kubernetes && popd
+          pushd ${BUILD_PATH}/src/k8s.io/kubernetes && make generated_files && popd
```

Comment:
I'm not sure about this but when we want to move to specific directory and only for that line we can use:
```
(cd ${BUILD_PATH}/src/k8s.io/kubernetes && make generated_files)
```

Reply:
I will look at the installation commands again.
Have I used the correct syntax to read and set environment variables in "actions" workflows?

Reply:
I think we should use `env.` eg `${{ env.BUILD_PATH} }`

- https://github.community/t/how-to-set-and-access-a-workflow-variable/17335
- https://docs.github.com/en/actions/reference/environment-variables
This should be inline with the one for [secret](https://github.com/kubernetes/website/pull/25771/files#diff-0f9a227f719fb077ec8062bccb5a9db75597db3a96d6a159be72cb52f4e506c7R13)

Reply:
OK. Thanks for the information.
I looked at: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable.
I can set BUILD_PATH in `jobs.build.env` assuming this variable has access to GITHUB_WORKSPACE. Alternatively, could create the installation at the workspace root and not use BUILD_PATH.
I set KUBE_VERSION in a step assuming that any later steps have access to this environment value as $KUBE_VERSION.